Package.swift

```diff
@@ -53,12 +52,11 @@ func dependencies() -> [PackageDescription.Package.Dependency] {
 }
 
 func mlxDependencies() -> [PackageDescription.Package.Dependency] {
-    let isMLXDisabled = ProcessInfo.processInfo.environment["MLX_DISABLED"] == "1"
-    if isMLXDisabled {
+    if isMLXDisabled() {
         return []
     } else {
         return [
-            .package(url: "https://github.com/ml-explore/mlx-swift", branch: "main"),
+            .package(url: "https://github.com/ml-explore/mlx-swift", revision: "d6d9472da5bf7ec2654e8914bd1d15622f45b6a9"),
```

Comment:
the latest version of `mlx` 0.16.0 breaks the implementation, let's pin it to the last commit before the breaking one, I'll have to talk to `mlx` team about the issue

Reply:
fixed, solution provided here https://github.com/ml-explore/mlx-swift/issues/117